ORStools/proc/directions_points_layer_proc.py

```diff
@@ -103,12 +105,18 @@ def processAlgorithm(self, parameters, context, feedback):
         )
 
         source_field_name = parameters[self.IN_FIELD]
+        if source_field_name:
+           get_fields_options = dict(
+                    from_type=source.fields().field(source_field_name).type(),
+                    from_name=source_field_name
+                )
+        else:
+
+        sink_fields = directions_core.get_fields(**get_fields_options, line=True)
```

Comment:
😅 did you actually test after adjusting?
-> indentation error 😝 
(pycharm would have noticed you....)

There is missing something here even if indented, if: defines `get_fields_options`, else: defines `sink_fields` ?
like this `get_fields_options` is not initialized if no `source_field_name` is given

Reply:
…I didn't :see_no_evil: 
Should be fixed :)